test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,177 @@ Feature: Bind an application to a service
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ClusterServiceVersion
+            metadata:
+                name: some-backend-service.v0.1.0
+            spec:
+                displayName: Some Backend Service
+                install:
+                    strategy: deployment
+                customresourcedefinitions:
+                    owned:
+                        - name: backservs.service.example.com
+                          version: v1
+                          kind: Backserv
+                          specDescriptors:
+                            - description: SVC name
+                              displayName: SVC name
+                              path: svcName
+                              x-descriptors:
+                                - binding:env:attribute
+
+            """
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The CR "demo-backserv-cr-1" for kind "Backserv" is present
+            """
+            apiVersion: service.example.com/v1
+            kind: Backserv
+            metadata:
+                name: demo-backserv-cr-1
+            spec:
+                svcName: demo-backserv-cr-1
+            """
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: sbr-csv-attribute
+            spec:
+                backingServiceSelector:
+                    group: service.example.com
+                    version: v1
+                    kind: Backserv
+                    resourceRef: demo-backserv-cr-1
+            """
+        Then Secret "sbr-csv-attribute" must contain data from resource type "Backserv" cr name "demo-backserv-cr-1" and jsonpath "{.spec.svcName}"
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "sbr-csv-attribute" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "sbr-csv-attribute" should be changed to "False"
+
+    Scenario: Create intermidiate secret using information mentioned in x-descriptors of backend service CSV
```

Comment:
what is the difference between this scenario and the previous one?

In both cases, binding definition is stored in x-descriptors.

I think we should call "binding secret" instead of "intermediate secret".

From user perspective, intermediate secrets are not important, i.e. they are not in focus of the use case. 

What user really wants to know is that a service managed by OLM operator can be bound to an application using binding definitions stored in OLM operator descriptor.

How about rephrasing to:

```
Binding secret gets created from the definitions managed in OLM operator descriptors
```

Reply:
> what is the difference between this scenario and the previous one?

The previous scenario is based on issue #656. So as it is a bug it's better to keep a test case for it.


> I think we should call "binding secret" instead of "intermediate secret".

We always have been calling it as an intermediate secret so I just continued it.

 
> 
> How about rephrasing to:
> 
> ```
> Binding secret gets created from the definitions managed in OLM operator descriptors
> ```

The name sounds cool.

Reply:
Fine.